### PR TITLE
SPOI-12043 README files

### PR DESCRIPTION
**Pull request by yogidevendra**
1. database-to-database-sync
2. hdfs-line-copy
3. hdfs-part-file-copy
4. kafka-to-hdfs-filter-transform

### CODE REVIEW

File: app-templates/kafka-to-hdfs-filter-transform/README.md [hunk 1]
```diff
@@ -1,52 +1,59 @@
...
+## Supported visualizations
+
+| Description  | Widget   |
+|---|---|---|---|---|
+| Bytes read per minute from Kafka  | Line chart|
+| Bytes written per minute to HDFS | Line chart |
+| Events read per minute from Kafka | Line chart |
+| Event read per minute to HDFS  | Line chart  |
+| Number of bad events per minute | Line chart  |
+| Total number of bad events | Line chart  |
```
Comment (chaithu14) [inline app-templates/kafka-to-hdfs-filter-transform/README.md]: Is line charts are available for bad events?

File: app-templates/kafka-to-hdfs-filter-transform/README.md [hunk 2]
```diff
@@ -1,52 +1,59 @@
...
+   ![Logical Plan](https://www.datatorrent.com/wp-content/uploads/2017/08/kafka_to_hdfs_filter_transform_dag.png)
+- It uses following operators
+  - Kafka single port input operator
+  - JSON parser
+  - Transform operator
+  - Filter operator
+  - JSON formatter
+  - String file output operator
+- Supported data source
+  - Apache Kafka version 0.9
```
Comment (chaithu14) [inline app-templates/kafka-to-hdfs-filter-transform/README.md]: Specify it as 0.9.0.1.

File: app-templates/hdfs-part-file-copy/README.md [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
+- Logical flow diagram
 
-&nbsp; <a href="http://docs.datatorrent.com/app-templates/hdfs-parallel-file-copy/"  class="docs" id="docs" ga-track="docs" target="_blank">http://docs.datatorrent.com/app-templates/hdfs-parallel-file-copy/</a>
+   ![Logical Plan](https://www.datatorrent.com/wp-content/uploads/2017/06/HDFS_Part_File_Copy_DAG.png)
+- It uses following operators
+  - FSInputModule
+  - PartFileWriter
+- Supported data source
+  - Apache Hadoop HDFS
+  - Tested with hadoop library: org.apache.hadoop:hadoop-common:jar:2.6.0
```
Comment (chaithu14) [inline app-templates/hdfs-part-file-copy/README.md]: I think these jars pick it at runtime. So, it depends on where you are running the apps.

Comment (yogidevendra) [inline app-templates/hdfs-part-file-copy/README.md]: Verified that we are using 2.6.0 on node36 cloudera deployment.

**Comment by chaithu14** [comment]
LGTM

File: app-templates/database-to-database-sync/README.md [hunk 4]
```diff
@@ -1,54 +1,51 @@
...
+## Description
+
+This application demonstrates continuous archival of big data from database tables.
+It first does a bulk upload from the configured table parallely and continuously polls at configured poll interval for new records and writes them in destination table.
+
 - The application scales linearly with the number of poller partitions.
 - The application is fault tolerant and can withstand node and cluster outages without data loss.
 - The application is also highly performant and can process as fast as the network allows.
 - It is extremely easy to add custom logic to get your business value without worrying about database connectivity and operational details of database poller and database writer.
 - The only configuration user needs to provide is source database connection details, table.
```
Comment (DT-Priyanka) [inline app-templates/database-to-database-sync/README.md]: please mention config needed source and destination

File: app-templates/hdfs-line-copy/README.md [hunk 5]
```diff
@@ -1,50 +1,54 @@
...
-Here is a preview of the properties to be set at application launch
+## Implementation details
 
-![Launch App Properties](http://datatorrent.com/wp-content/uploads/2016/11/HDFS_HDFS_Line_Copy_properties.png)
+- Logical flow diagram
 
-### Resources
+   ![Logical Plan](http://datatorrent.com/wp-content/uploads/2016/11/HDFS_HDFS_Line_Copy_DAG.png)
+- It uses following operators
+  - FSRecordReaderModule
```
Comment (DT-Priyanka) [inline app-templates/hdfs-line-copy/README.md]: Not sure, do we give links to these operators (In malhar or moodi repository)

**Pull request merged**